modules/store/src/reducer_manager.ts

```diff
@@ -26,6 +25,10 @@ export const UPDATE = '@ngrx/store/update-reducers' as const;
 export class ReducerManager
   extends BehaviorSubject<ActionReducer<any, any>>
   implements OnDestroy {
+  get reducerSnapshot(): ActionReducerMap<any, any> {
```

Comment:
I'd rather not use `snapshot` here

```suggestion
  get currentReducers(): ActionReducerMap<any, any> {
```

Reply:
Thanks for the suggestion.
I will apply it.

I came up with snapshot, since Angular uses this term, too (see ActivatedRoute) and I wanted to reuse already known terms.

But I am fine with currentReducers, too.